<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Image Data</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="editImage">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonOpenImage">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="toolGroupBox">
          <property name="title">
           <string>Tool</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Pen width:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinPenWidth">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Labeling</string>
          </property>
          <layout class="QVBoxLayout" name="_2">
           <item>
            <widget class="QTreeWidget" name="treeLabels">
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerMinimumSectionSize">
              <number>5</number>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>2</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Segmentation</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="buttonLoadModel">
             <property name="text">
              <string>Load Model ...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonSaveModel">
             <property name="text">
              <string>Save Model ...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonTrain">
             <property name="text">
              <string>Train</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonApply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>444</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progress">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="DrawingPlotWidget" name="preview" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionBrush"/>
   <addaction name="actionErase"/>
  </widget>
  <action name="actionBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/icons/pencil.png</normaloff>
     <disabledoff>:/icons/brush_dark_dis.png</disabledoff>:/icons/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/icons/draw_eraser.png</normaloff>
     <disabledoff>:/icons/eraser_dark.png</disabledoff>:/icons/draw_eraser.png</iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/icons/folder.png</normaloff>
     <disabledoff>:/icons/folder_dark_dis.png</disabledoff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/icons/disk.png</normaloff>
     <disabledoff>:/icons/save_dark_dis.png</disabledoff>:/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save ...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingPlotWidget</class>
   <extends>QWidget</extends>
   <header>DrawingPlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
